src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -118,10 +159,12 @@ public List<Artist> SearchForNewArtist(string title)
 
                 SetCustomProvider();
 
-                var httpRequest = customerRequestBuilder.Create()
+                var httpRequest = _customerRequestBuilder.Create()
                                     .SetSegment("route", "search")
                                     .AddQueryParam("type", "artist")
                                     .AddQueryParam("query", title.ToLower().Trim())
+                                    //.AddQueryParam("images","false") // Should pass these on import search to avoid looking to fanart and wiki 
```

Comment:
Do we want to go ahead and enable these on metadata or wait a bit?

Reply:
I need to find a way to only pass them on an import and not a regular artist add (we like to have images and overviews for that)